src/coreComponents/constitutive/contact/ContactBase.hpp

```diff
@@ -247,7 +247,8 @@ GEOS_FORCE_INLINE
 real64 ContactBaseUpdates::computeHydraulicAperture( real64 const aperture,
                                                      real64 & dHydraulicAperture_dAperture ) const
 {
-  return m_apertureTable.compute( &aperture, &dHydraulicAperture_dAperture );
+  dHydraulicAperture_dAperture = 1;
+  return aperture; 
```

Comment:
I don't think we won't to allow for 0 hydraulic aperture anyways. In a contact problem it would result in a row of zeros in case of closed fracture. I think that in the other solver I simply added a constant that defines minimum value for the hydraulic aperture https://github.com/GEOS-DEV/GEOS/blob/958e393cc4633b8cd97f830b655a1c2106fde918/src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsFractures.hpp#L72

This should not really matter for quasi-newton though. right?

Reply:
Yes, I don't think adding artificial aperture matters Q-N. But in the KGD test, the performance of QN is a bit different (worse) than the code I tested several months ago. In future, we can test the sensitivity of minimumHydraulicAperture to the nonlinear solver. BTW, is the minimumHydraulicAperture assigned from "defaultAperture" in input xml?

Reply:
@Guotong-Ren I removed that code change. First - there could be a numerical problem with zero row that Matteo mentioned, and second - hydraulic aperture responsible for some physics, right? So we better keep the mapping there.